tests/baselines/reference/inlayHintsQuotePreference2.baseline

```diff
@@ -0,0 +1,33 @@
+export function fn(a = a1, b = b1) {}
+                    ^
+{
+  "text": "",
+  "displayParts": [
+    {
+      "text": ": "
+    },
+    {
+      "text": "'''"
```

Comment:
Hmmm using the quote preference here means that this is no longer a single quote literal, and I wonder if this might end up confusing people in some cases. @DanielRosenwasser wdyt about this?

Reply:
You need to still use `escapeString`

Reply:
@a-tarasyuk I think you need to escape the string after quoting it. Notice how this is still `'''` which is not a single quote literal.